service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -64,6 +69,14 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
+        List<Element<HearingBooking>> element = defaultIfNull(
+            caseData.getHearingDetails(), wrapElements(HearingBooking.builder().build()));
+
+        String hearingDate = getHearingDate(element);
+
+        Stream.of(DirectionAssignee.values()).forEach(assignee ->
```

Comment:
Could you use the same field across multiple pages (think not but can't remember)?

Reply:
no, you get a unique constraint violation exception 😕

Reply:
Thought so 😢